apps/browser/qml/pages/components/ConfigDialog.qml

```diff
@@ -98,26 +99,50 @@ Dialog {
             id: loader
             height: Theme.itemSizeMedium
             width: prefsList.width
-            sourceComponent: model.type === 128 ? textSwitch : textField
+            sourceComponent: model.type == WebEngineSettings.BoolPref ? textSwitch : textField
 
             Component {
                 id: textField
 
                 TextField {
-                    label: model.name
+                    readonly property bool digitsOnly: model.type == WebEngineSettings.IntPref
+
+                    label: {
+                        if (errorHighlight && digitsOnly) {
+                            if (text.charAt(0) == "-") {
+                                //% "Select a value that is no less than -2147483648"
+                                return qsTrId("sailfish_browser-la-min_int_warning")
+                            }
+
+                            //% "Select a value that is no more than 2147483647"
+                            return qsTrId("sailfish_browser-la-max_int_warning")
```

Comment:
In practice, the `errorHighlight` seems to cover more cases than just bounding (e.g. entering a decimal like "0.01" or paste a string "abcd"). It might be safer just to have error text "Please enter only integer values" or something like that.

Reply:
Fine for me.

Reply:
Also description does work nicely if you're wondering.